planemo/galaxy/config.py

```diff
@@ -761,7 +761,7 @@ def install_workflows(self):
                 self._install_workflow(runnable)
 
     def _install_workflow(self, runnable):
-        if self._kwds.get("shed_install"):
+        if self._kwds.get("shed_install") and self._kwds.get("engine") != "external_galaxy":
```

Comment:
or maybe something like: `(self._kwds.get("engine") != "external_galaxy" and not self._kwds.get("galaxy_admin_key"))`

so it would still be possible to run shed_install on an external galaxy instance, as long as the admin key was specified.

Reply:
> so it would still be possible to run shed_install on an external galaxy instance, as long as the admin key was specified.

I think this is an important use-case, e.g. for a Galaxy test server. Or a training instance that will be setup.